.scala-steward.conf

```diff
@@ -1,5 +1,5 @@
 updates.pin = [
-  { groupId = "com.fasterxml.jackson.core", version = "2.16." }
+  { groupId = "com.fasterxml.jackson.core", version = "2.17." }
```

Comment:
do we need this at all now?

Reply:
I think it is best to keep them for now. Upgrades in jackson have proved to be complicated in the past.